src/auth/authRouter.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+import User from './model';
+import express from 'express';
+
+const Authrouter = express.Router();

```

Comment:
I think the Authrouter should be lowercase to be consistent with our other router